shell/platform/fuchsia/dart_runner/dart_test_component_controller_v2.cc

```diff
@@ -48,7 +48,7 @@ namespace dart_runner {
 namespace {
 
 constexpr char kTmpPath[] = "/tmp";
-constexpr char kTestCaseName[] = "dart_test_v2";
```

Comment:
This is no longer a constant so we shouldn't write it like a constant. Can we make it a field on the class instead?

Reply:
Looks like nested classes don't have access to the member variables of the enclosing class. AFAICT this is the way to relay the test name info from the DartTestComponentController to the CaseIterator class.

Reply:
We define the DartTestComponentControllerV2::CaseIterator class and its constructor so we can also add whatever data we want into it, such as `std::string case_name_`.

Reply:
Gotcha